infrastructure/template.json

```diff
@@ -0,0 +1,183 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serverfarms_ASP_test_bfbf_name": {
+            "defaultValue": "ASP-test-bfbf",
+            "type": "String"
+        },
+        "sites_TechnicalDocuIndexerWeb_name": {
+            "defaultValue": "TechnicalDocuIndexerWeb",
+            "type": "String"
+        },
+        "components_TechnicalDocuIndexerWeb_name": {
+            "defaultValue": "TechnicalDocuIndexerWeb",
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "microsoft.insights/components",
+            "apiVersion": "2018-05-01-preview",
+            "name": "[parameters('components_TechnicalDocuIndexerWeb_name')]",
+            "location": "westeurope",
```

Comment:
Location should be parametrized as a variable for example: `"location": "[resourceGroup().location]"`

Reply:
parametrized location